plugins/modules/win_get_url.py

```diff
@@ -125,6 +131,13 @@
     checksum: a97e6837f60cec6da4491bab387296bbcd72bdba
     checksum_algorithm: sha1
     force: true
+
+- name: Download src with set_zone on
```

Comment:
Are you able to change with set_zone on to some a bit more descriptive, like download src and add the mark of the web or something along those lines?